requirements.txt

```diff
@@ -26,12 +28,19 @@ multidict==6.0.5
     # via
     #   aiohttp
     #   yarl
+packaging==24.2
+    # via pytest
+pluggy==1.5.0
+    # via pytest
 py-cord==2.6.1
 pydantic==2.7.1
     # via pydantic-settings
 pydantic-core==2.18.2
     # via pydantic
 pydantic-settings==2.2.1
+pytest==8.3.4
```

Comment:
best practice is to separate out your test deps

Reply:
agreed, but also i don't wannaaaaaa

Reply:
too bad

Reply:
when you're right you're right. much cleaner now